servlet-engine/src/main/java/io/micronaut/servlet/engine/DefaultServletHttpHandler.java

```diff
@@ -33,32 +37,48 @@
  */
 @Singleton
 public class DefaultServletHttpHandler extends ServletHttpHandler<HttpServletRequest, HttpServletResponse> {
+    private final Executor ioExecutor;
+
     /**
      * Default constructor.
      *
      * @param applicationContext The application context
      * @param conversionService  The conversion service
+     * @param ioExecutor         Executor to use for blocking IO operations
      */
-    public DefaultServletHttpHandler(ApplicationContext applicationContext, ConversionService conversionService) {
+    public DefaultServletHttpHandler(ApplicationContext applicationContext, ConversionService conversionService, @Named(TaskExecutors.BLOCKING) Executor ioExecutor) {
```

Comment:
these should seems specific to Servlet, how will this work with GCP/AWS where we have the failing tests now?

Reply:
I searched for DefaultServletHttpRequest in micronaut-projects and this was the only usage, so I assume GCP/AWS still go through this handler